Recrutify/Recrutify.Host/Controllers/OData/ProjectsController.cs

```diff
@@ -28,5 +29,17 @@ public IQueryable<ProjectDTO> Get()
         {
             return _projectService.Get();
         }
+
+        [EnableQuery(
+            HandleNullPropagation = HandleNullPropagationOption.False,
+            AllowedQueryOptions = AllowedQueryOptions.Filter | AllowedQueryOptions.OrderBy | AllowedQueryOptions.Top | AllowedQueryOptions.Skip | AllowedQueryOptions.Count)]
+        public IEnumerable<ShortProjectDTO> GetAllProject(ODataQueryOptions<ShortProjectDTO> options)
+        {
+            var candidates = _projectService.Get();
+            var filteredCandidates = options.ApplyTo(candidates) as IEnumerable<ShortProjectDTO>;
+            var result = filteredCandidates!.Where(x => x.IsActive = true).Where(x => x.StartDate >= DateTime.Now)
+                                            .OrderBy(x => x.StartDate);
```

Comment:
StartRegistrationDate everywhere

Reply:
done

Reply:
StartRegistrationDate

Reply:
IsActive==true

Reply:
Only one Where should be with conditions via &&

Reply:
done

Reply:
Orderby StartRegistrationDate